app/serializers/trip_serializer.rb

```diff
@@ -12,7 +12,12 @@
 #
 
 class TripSerializer < ActiveModel::Serializer
-  attributes :id, :name, :description, :created_at, :updated_at
+  attributes :id, :name, :description, :created_at, :updated_at, :user_id, :traveller_name
 
   has_many :cities
+
+  def traveller_name
+    User.find(object.user_id).name
```

Comment:
@zeus999 can we please change this `name` to `full_name` a method and for now, it can just return the attribute name only but very soon we'll have attributes like `first_name` & `last_name` so that is where it helps and then we won't have to make many changes.